src/test/CMakeLists.txt

```diff
@@ -57,7 +57,6 @@ set(engine_tests
 	   unit/engine/InputTest.cpp
 	   unit/engine/LinkTest.cpp
 	   unit/engine/NetworkTest.cpp
-	   unit/engine/YAMLUtilsTest.cpp
```

Comment:
> is the new ValueMapTest file (below) being built?

Yes but it is not complete.  It will eventually combine tests that were in YamalUtilsTest, ScalerTests, and ValueTests.  Since it no longer does the parsing with Spec, some of the tests no longer apply.

Reply:
I was asking since I didn't see the file ValueMapTest in the Cmakelists

Reply:
Oh,  it will be in ValueTest.cpp